core/src/main/scala/org/apache/pekko/persistence/jdbc/query/dao/DefaultReadJournalDao.scala

```diff
@@ -60,6 +60,12 @@ class DefaultReadJournalDao(
   override def journalSequence(offset: Long, limit: Long): Source[Long, NotUsed] =
     Source.fromPublisher(db.stream(queries.journalSequenceQuery((offset, limit)).result))
 
+  override def lastPersistenceIdSequenceNumber(
+      persistenceId: String
```

Comment:
Could you use the existing code style? See the function above as an example.

Reply:
Sure! Tried to align to existing code style in <https://github.com/apache/pekko-persistence-jdbc/pull/267/commits/49b7262dbcfe47279d580537a7c65dcdfd460fc8>.